packages/react-api/__tests__/hooks/useCartoLayerFilterProps.test.js

```diff
@@ -1,15 +1,16 @@
 import { DataFilterExtension } from '@deck.gl/extensions';
 import { renderHook } from '@testing-library/react-hooks';
 import useCartoLayerProps from '../../src/hooks/useCartoLayerProps';
```

Comment:
The name of this test file is OLD (LayerFilterProps...) and has to change

Reply:
Good catch!